src/WebJobs.Script/Description/Binding/BindingMetadata.cs

```diff
@@ -40,25 +47,11 @@ public bool IsTrigger
         {
             get
             {
-                return
-                    Type == BindingType.TimerTrigger ||
-                    Type == BindingType.BlobTrigger ||
-                    Type == BindingType.HttpTrigger ||
-                    Type == BindingType.QueueTrigger ||
-                    Type == BindingType.EventHubTrigger ||
-                    Type == BindingType.ServiceBusTrigger ||
-                    Type == BindingType.ManualTrigger ||
-                    Type == BindingType.ApiHubFileTrigger;
+                return Type.EndsWith("trigger", StringComparison.OrdinalIgnoreCase);
             }
         }
 
-        /// <summary>
-        /// Bindings can include information that drives the JobHostConfiguration.
-        /// </summary>
-        /// <param name="configBuilder"></param>
-        public virtual void ApplyToConfig(JobHostConfigurationBuilder configBuilder)
-        {
-            // default is nop
-        }        
+        // TEMP
+        public JObject Raw { get; set; }     
```

Comment:
This entire BindingMetadata class is in flux. Likely there will be further refactorings in this area. I slapped Raw on here because that's the info the new extension provider model needs (it doesn't reference this BindingMetadata type).